monkey/monkey_island/cc/services/utils/file_permissions.py

```diff
@@ -0,0 +1,8 @@
+import os
+
+
+def has_sufficient_permissions(path: str, required_permissions: str) -> bool:
```

Comment:
Maybe `has_expected_permissions()` would be a better name? You could then change `required_permissions` to `expected_permissions`. I'm not married to that name, but I don't think `sufficient` is the correct adjective.